packages/console/src/pages/ConnectorDetails/index.tsx

```diff
@@ -113,6 +116,14 @@ const ConnectorDetails = () => {
             <div>
               <ConnectorTypeName type={data.type} />
               <div className={styles.verticalBar} />
+              {connectorFactory && (
+                <>
+                  <div className={styles.factoryName}>
```

Comment:
is this factory or instance?

Reply:
is factory: `OIDC`(factory) -> `Google`(instance)